Atomic/atomic.py

```diff
@@ -671,7 +667,6 @@ def get_label(label):
             version = "%s-%s-%s" % (current_name, get_label("Version"),
                                     get_label("Release"))
```

Comment:
variable `version` is assigned but never used, should there be a reason to keep it?